tests/components/devolo_home_control/test_init.py

```diff
@@ -0,0 +1,68 @@
+"""Tests for the devolo Home Control integration."""
+from devolo_home_control_api.exceptions.gateway import GatewayOfflineError
+import pytest
+
+from homeassistant.components.devolo_home_control import (
+    async_setup_entry,
```

Comment:
Use the core interfaces to start the tests.

Reply:
Done. It obviously matters from which other integration you copy ;)